mozci/console/commands/push.py

```diff
@@ -341,6 +349,51 @@ def handle(self) -> None:
             self.log_pushes(PushStatus.UNKNOWN, True),
         ]
 
+        if self.option("detailed-classifications"):
+            self.line("\n")
+
+            real_stats = [0, 0, 0, 0]
+            intermittent_stats = [0, 0, 0, 0]
+            for push in self.pushes:
+                if self.failures.get(push) and (
+                    self.failures[push]["real"] or self.failures[push]["intermittent"]
```

Comment:
Currently, it's skipped because we would just print `Printing detailed classifications comparison for push {push.branch}/{push.rev}` without anything else after since there is nothing to compare. So I don't understand why we should remove the condition

Reply:
You're right, we should change this a bit. If the algorithm returns no results, we still need to take that into account.

Right now we are only checking that what the algorithm returns is correct, but we should also check that it is complete.

We can do this in a follow-up though, so let's keep it as it is for now. I will file a follow-up issue for the rest, and it is less important.

Reply:
Filed https://github.com/mozilla/mozci/issues/637.